src/core/sorted_map.cc

```diff
@@ -108,7 +108,8 @@ constexpr unsigned kConvFlags = DoubleToStringConverter::UNIQUE_ZERO;
 DoubleToStringConverter score_conv(kConvFlags, "inf", "nan", 'e', -6, 21, 6, 0);
 
 // Copied from redis code but uses double_conversion to encode double values.
-unsigned char* zzlInsertAt(unsigned char* zl, unsigned char* eptr, sds ele, double score) {
+unsigned char* zzlInsertAt(unsigned char* zl, unsigned char* eptr, const unsigned char* ele,
```

Comment:
nit: I would use `const char* ele` to avoid multiple castings on the caller side.
Maybe even better: it's now in our c++ code base, so you could even keep `string_view  ele` here and make the safe conversion to `""sv` in this function.

Reply:
btw, to signify it's ours - please rename it to `ZzlInsertAt`

Reply:
changed name and signature